src/codeu/chat/server/Model.java

```diff
@@ -125,4 +129,19 @@ public void add(Message message) {
   public StoreAccessor<String, Message> messageByText() {
     return messageByText;
   }
+
+  public void add(Uuid userid, Uuid interest) {
```

Comment:
This is just following the format of the previous adds, they are all just named add.